src/main/proto/spine/options.proto

```diff
@@ -1093,3 +1122,81 @@ message IfSetAgainOption {
     //
     string error_msg = 1;
 }
+
+// Defines the error message used if a `distinct` field has duplicates.
```

Comment:
It would be good to mention which objects we consider to be duplicates.

This is more challenging than it seems, because some objects, like Entities, have IDs. And users of this API might think that could be some special case: comparing by IDs only. Which, of course, it isn't.